CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 
 # PayPal iOS SDK Release Notes
 
+## unreleased
+* PaymentButtons
+  * Add new size `.miniWithWordmark`
+  * Add new labels, such as `.addMoneyWith`, `.bookWith`, and `buyNowWith` for `PayPalButton.Label`
```

Comment:
We should call out the enum cases for `PaymentButtonLabel` we removed (`.checkout` and `.buyNow`) since those removals will break a merchant app.

Nit - but I would also include the full list of the enum cases added, even though it's long

Reply:
By break a merchant app, I mean if they were to upgrade to V2 and their code was:

```
PayPalButton.Representable(
    color: .black,
    edges: .softEdges,
    size: .expanded,
    label: .buyNow
)
```

it would not compile since the `.buyNow` option is gone

Reply:
Good point! Thanks, I'll make the changes